x/stake/keeper/key.go

```diff
@@ -34,38 +34,40 @@ var (
 
 const maxDigitsForAccount = 12 // ~220,000,000 atoms created at launch
 
-// get the key for the validator with address.
+// GetValidatorKey Gets the key for the validator with address.
 // VALUE: stake/types.Validator
 func GetValidatorKey(ownerAddr sdk.AccAddress) []byte {
 	return append(ValidatorsKey, ownerAddr.Bytes()...)
 }
 
-// get the key for the validator with pubkey.
+// GetValidatorByPubKeyIndexKey Gets the key for the validator with pubkey.
 // VALUE: validator owner address ([]byte)
 func GetValidatorByPubKeyIndexKey(pubkey crypto.PubKey) []byte {
 	return append(ValidatorsByPubKeyIndexKey, pubkey.Bytes()...)
 }
 
-// get the key for the current validator group
+// GetValidatorsBondedIndexKey Gets the key for the current validator group.
```

Comment:
nah - this is overly redundant remove this added superfluousness from this comment and the above three - again Tendermint Lint doesn't require this - you should upgrade your linter if it's not already